e2e/playwright/folder-rules/src/tests/create-rules.e2e.ts

```diff
@@ -0,0 +1,319 @@
+/*!
+ * Copyright © 2005-2024 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Alfresco Example Content Application
+ *
+ * This file is part of the Alfresco Example Content Application.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail. Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * The Alfresco Example Content Application is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The Alfresco Example Content Application is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * from Hyland Software. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { expect } from '@playwright/test';
+import { ActionType, ApiClientFactory, test, Utils, TrashcanApi, NodesApi, Comparator, Field } from '@alfresco/playwright-shared';
+
+test.use({ launchOptions: { slowMo: 300 } });
+test.describe('Folder Rules Actions', () => {
+  const apiClientFactory = new ApiClientFactory();
+  let nodesApi: NodesApi;
+  let trashcanApi: TrashcanApi;
+  const username = `user-e2e-${Utils.random()}`;
+  const folderName883 = `folder-XAT-883-${Utils.random()}`;
```

Comment:
Do we really need that many different folders? Maybe some cases can be modified to reduce this need, it looks scary to me

Reply:
Yes, we need that many different folders. Either I make that many different folders OR the tests themselves would be much bigger.

Its because many locators are shared between the rules component and create rule component, which makes it hard to use these locators in tests, thats why a clean folder is a easy and fast way to avoid this type of issues.

I tried to use locators with getChild so it doesnt take the locators hidden behind the create rule component, but it didnt seem to work in all cases so I decided to just stay with seperate folders.

I thought about making a seperate file just for creating these folders and exporting them to the test file but I wasnt sure if I want to spend time on it as it can be tricky to do

Reply:
I reduced the number of folders in _create-rules.e2e.ts_ in my previous PR. 

I will do the same for _update-rules.e2e.ts_ in this PR.

Reply:
Wonderful, thanks!